frontend/src/components/pages/DeletionJob.js

```diff
@@ -207,45 +211,25 @@ const DeletionJob = ({ gateway, jobId }) => {
               <Icon type={`alert-${successJobClass(job.JobStatus)}`} />
               <span>{job.JobStatus}</span>
             </DetailsBox>
-            <DetailsBox label="Deletion Queue Size">
-              {job.DeletionQueueItems.length}{" "}
-              {job.DeletionQueueItems.length > 0 && (
-                <>
-                  (
-                  <Button
-                    variant="link"
-                    style={{ padding: 0 }}
-                    onClick={() => showDeletionQueue(true)}
-                  >
-                    View Deletion Queue
-                  </Button>
-                  )
-                </>
-              )}
-            </DetailsBox>
-            <DetailsBox
-              label="Deletion Queue Items Skipped"
-              className={`status-label ${warningCountClass(
-                job.DeletionQueueItemsSkipped
-              )}`}
-            >
-              <Icon
-                type={`alert-${warningCountClass(
-                  job.DeletionQueueItemsSkipped
-                )}`}
-              />
-              <span>
-                {job.DeletionQueueItemsSkipped.toString().toUpperCase()}
-              </span>
+            <DetailsBox label="Deletion Queue Size" fullWidth>
+              {isUndefined(job.DeletionQueueSize)
+                ? syncQueueProcessing(job)
+                  ? job.DeletionQueueItems.length
+                  : "Unknown (Planning pending)"
```

Comment:
```suggestion
                  : "Unknown (Plan pending)"
```

Re: https://github.com/awslabs/amazon-s3-find-and-forget/pull/239#issuecomment-789836706

If the queue size is determined from the product of planning, then that'd make it a plan that we're awaiting on.